### PR TITLE
create quote brand component

### PR DESCRIPTION
**Pull request by rogervera**
## Create a Quote component for the Brand Package

Will be used in creating the Learner Testimonials component
Screen Shot 2020-02-20 at 5 13 01 PM

Screen Shot 2020-02-20 at 11 47 07 AM

---

## Merging your changes

When you are ready to merge to master, use the "squash and merge" button in GitHub, and follow the [commit message guide](/README.md#commit-message-guide) to write your commit message.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.2-alpha.afd728.4+afd7284d<br/>@codecademy/gamut-templates@2.2.33-alpha.afd728.4+afd7284d<br/>@codecademy/gamut@2.4.9-alpha.afd728.4+afd7284d<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.2-alpha.7fd720.5+7fd7206c<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.2.1-alpha.c6334e.7+c6334eb7<br/></pre>

File: packages/brand-components/src/Quote/__tests__/Quote-test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    const containerClassName = wrapper.find(`div`).prop('className');
+
+    expect(containerClassName).toContain(styles.lightContainer);
+  });
+
+  it('adds the light class to the container when its theme is light', () => {
+    const wrapper = mount(
+      <Quote
+        text="The reason I will not exhibit this picture is that I am afraid that I have shown in it the secret of my own soul."
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Quote/__tests__/Quote-test.tsx]: Do I get any bonus points for all the high brow intellectual references you put in these?

File: packages/brand-components/src/Quote/__tests__/Quote-test.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { VisualTheme } from '../../../../gamut/src/theming/VisualTheme';
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Quote/__tests__/Quote-test.tsx]: Just noticed this now (same with `Avatar`) - I wonder if this'll still work when imported as a node module? I was using `@codecademy/gamut` in the GridForm PR (#605).

Comment (rogervera) [reply packages/brand-components/src/Quote/__tests__/Quote-test.tsx]: Great point, to be on the safe side I just changed the imports
https://github.com/Codecademy/client-modules/pull/635/commits/9a3ad33ee0ba10045b008c5a7ea07a98c4e37ea0

Comment (rogervera) [reply packages/brand-components/src/Quote/__tests__/Quote-test.tsx]: Will test both alpha versions in monolith right now for knowledge's sake.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.2.1-alpha.9a3ad3.8+9a3ad33e<br/></pre>

File: packages/brand-components/src/Quote/__tests__/Quote-test.tsx (hunk 2 above)
Comment (rogervera) [reply packages/brand-components/src/Quote/__tests__/Quote-test.tsx]: @JoshuaKGoldberg 

Screen Shot 2020-02-21 at 10 28 50 AM
Screen Shot 2020-02-21 at 10 39 24 AM
Confirmed `'../../../../gamut/src/theming/VisualTheme';` will RUIN EVERYTHING

**Pull request merged**